context/app/static/js/components/savedLists/SavedEntitiesTable/SavedEntitiesTable.jsx

```diff
@@ -64,7 +64,7 @@ function SavedEntitiesTable({ savedEntities, deleteCallback, setShouldDisplaySav
           </BottomAlignedTypography>
         }
         buttons={
-          selectedRowsSize > 0 && (
+          selectedRowsSize === 0 ? null : (
```

Comment:
What does this fix?

Reply:
Returning `false` is a prop-type error, but `null` is fine. Maybe there's a better idiom?